spaces/models.py

```diff
@@ -0,0 +1,146 @@
+from users.models import Host
+
+class Space(models.Model):
+    host         = models.ForeignKey('users.Host', on_delete=models.CASCADE)
+    name         = models.CharField(max_length=100)
+    price        = models.DecimalField(max_digits=18 ,decimal_places=2)
+    description  = models.CharField(max_length=3000)
+    place_type   = models.ForeignKey('spaces.PlaceType', on_delete=models.CASCADE)
+    sub_property = models.ForeignKey('spaces.SubProperty', on_delete=models.CASCADE)
+    rating       = models.DecimalField(max_digits=3, decimal_places=2, null=True)
+    pub_date     = models.DateField(auto_now_add=True)
+
+    class Meta:
+        db_table = 'spaces'
+
+
+class PlaceType(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'place_types'
+
+
+class Property(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'properties'
+
+
+class SubProperty(models.Model):
+    space_property = models.ForeignKey('spaces.Property', on_delete=models.CASCADE) 
+    name           = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'sub_properties'
+
+
+class Option(models.Model):
+    space    = models.ForeignKey('spaces.Space', on_delete=models.CASCADE)
+    name     = models.CharField(max_length=45)
+    icon_url = models.URLField(max_length=2000)
+
+    class Mata:
+        db_table = 'options'
+
+
+class SpaceOption(models.Model):
+    space  = models.ForeignKey('spaces.Space', on_delete=models.CASCADE)
+    option = models.ForeignKey('spaces.Option', on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'space_options'
+
+
+class Location(models.Model):
+    space          = models.ForeignKey('spaces.Space', on_delete=models.CASCADE)
+    country        = models.ForeignKey('users.Country', on_delete=models.CASCADE)
+    region         = models.CharField(max_length=45)
+    city           = models.CharField(max_length=20)
+    address        = models.CharField(max_length=150)
+    address_detail = models.CharField(max_length=45)
+    letitude       = models.DecimalField(max_digits=10, decimal_places=6,null=True)
+    longitude      = models.DecimalField(max_digits=10, decimal_places=6,null=True)
+
+    class Meta:
+        db_table = 'locations'
+
+
+class Image(models.Model):
+    space     = models.ForeignKey('spaces.Space', on_delete=models.CASCADE)
+    image_url = models.URLField(max_length=2000)
+
+    class Meta:
+        db_table = 'images'
+
+
+class Detail(models.Model):
+    space      = models.ForeignKey('spaces.Space', on_delete=models.CASCADE)
+    max_people = models.IntegerField(default=0)
+    bathroom   = models.IntegerField(default=0)
+    
+    class Meta:
+        db_table = 'details'
+
+
+class Bedroom(models.Model):
+    space   = models.ForeignKey('spaces.Space', on_delete=models.CASCADE)
+    bedroom = models.IntegerField()
+
+    class Meta:
+        db_table = 'bedrooms'
```

Comment:
`침실` 데이터를 관리하는 테이블로 잘 만들어 주셨습니다.
`1 번 침실` \ `2 번 침실` \  `공용 공간` 과 같은 데이터가 저장되기 때문에, 숫자를 가지고 있는 것이 아니라 `1 번 침실` \ `2 번 침실` \  `공용 공간` 데이터를 저장할 수 있는 name field가 필요합니다.
```suggestion
class Bedroom(models.Model):
    name    = models.CharField()
    space   = models.ForeignKey('spaces.Space', on_delete=models.CASCADE)

    class Meta:
        db_table = 'bedrooms'
```

Reply:
추가로, 침실의 개수는 space를 참조하고 있는 bedroom의 개수를 count하면 되기 때문에 bedroom의 개수를 의미하는 필드를 만들 필요가 없습니다.

Reply:
넵!! 수정하겠습니다~